app/helpers/collection_resources_helper.rb

```diff
@@ -0,0 +1,13 @@
+module CollectionResourcesHelper
+  def metadata_helper(file, pages = nil)
+    extension = file.split(".")[-1]
+    type_string = case extension
+                  when "xlsx" then "Microsoft Excel"
+                  when "pdf" then "PDF"
+                  else extension
+                  end
+    size = number_to_human_size(File.size("public/files/#{file}"), precision: 0, significant: false)
+    pages_string = pages.present? ? pluralize(pages, "page") : nil
+    [type_string, size, pages_string].compact.join(", ")
+  end
```

Comment:
Can you think of a name that describes what it does? (e.g., it probably shouldn't include `_helper`) 

```suggestion
  def file_type_size_and_pages(file, number_of_pages: nil)
    extension_mapping = {
      ".xlsx" => "Microsoft Excel",
      ".pdf"  => "PDF",
    }
    extension = File.extname(file)[1..-1]
    
    file_type = extension_mapping.fetch(extension, extension)
    file_size = number_to_human_size(File.size("public/files/#{file}"), precision: 0, significant: false)
    file_pages = number_of_pages ? pluralize(number_of_pages, "page") : ""
    
    [file_type, file_size, file_pages].join(", ")
  end
```

Reply:
sure, good point